src/main/java/com/softserveinc/ita/rozetka/page_objects/RozetkaHeaderPage.java

```diff
@@ -0,0 +1,17 @@
+package com.softserveinc.ita.rozetka.page_objects;
+
+import static com.codeborne.selenide.Selenide.$x;
+import static com.codeborne.selenide.Selenide.open;
+
+public class RozetkaHeaderPage {
```

Comment:
rename to RozetkaHeader or Header, I guess this is not a page, it is part of page

Reply:
Page Object name must end on 'Page'